satellite/gracefulexit/endpoint_test.go

```diff
@@ -232,6 +239,90 @@ func TestConcurrentConnections(t *testing.T) {
 	})
 }
 
+func TestRecvTimeout(t *testing.T) {
+	testplanet.Run(t, testplanet.Config{
+		SatelliteCount:   1,
+		StorageNodeCount: 9,
+		UplinkCount:      1,
+		Reconfigure: testplanet.Reconfigure{
+			NewStorageNodeDB: func(index int, db storagenode.DB, log *zap.Logger) (storagenode.DB, error) {
```

Comment:
Maybe we don't need to rely on storagenode timeout to test satellite timeout? We probably can call the `Process` endpoint manually and then sleep for more then the receive timeout to receive the timeout message on the storagenode side?